app/src/main/java/me/saket/dank/ui/preferences/adapter/LookAndFeelPreferencesConstructor.java

```diff
@@ -70,15 +73,23 @@ public LookAndFeelPreferencesConstructor(
           }));
     }
 
-    uiModels.add(UserPreferenceSectionHeader.UiModel.create(c.getString(R.string.userprefs_group_subreddit)));
-
-    uiModels.add(UserPreferenceSwitch.UiModel.create(
-        c.getString(R.string.userprefs_submission_thumbnails),
-        showSubmissionThumbnails.get()
-            ? c.getString(R.string.userprefs_submission_thumbnail_summary_on)
-            : c.getString(R.string.userprefs_submission_thumbnail_summary_off),
-        showSubmissionThumbnails.get(),
-        showSubmissionThumbnails));
+      uiModels.add(UserPreferenceSectionHeader.UiModel.create(c.getString(R.string.userprefs_group_subreddit)));
+
+      uiModels.add(UserPreferenceSwitch.UiModel.create(
+              c.getString(R.string.userprefs_submission_thumbnails),
```

Comment:
Could you fix indentation here and in other places? Project uses 2 spaces for indentation (and 4 spaces for continuing indent) everywhere and I would like to keep this consistent. 

Note to self: Add shared Android Studio code style settings to repository.